src/components/appBar/CyVerseAppBar.js

```diff
@@ -172,7 +172,7 @@ function CyverseAppBar(props) {
                             >
                                 <AccountCircleIcon
                                     fontSize="large"
-                                    style={{ color: palette.white }}
+                                    style={{ color: theme.palette.white }}
```

Comment:
I'm not sure what to do about this color, but I wonder if `theme.palette.primary.contrastText` is meant for a situation like this. 🤔

Reply:
Seems like the default for 'theme.palette.primary.contrastText` is white, but not sure if that gets recalculated once we modify `theme.palette.primary` with CyVerse colors.  It sounds like the right one to use anyway: https://material-ui.com/customization/default-theme/

Reply:
Yup, it gets recalulated. It's showing up gray instead of white. Working on a fix now.

Reply:
Wait, no. I'm confusing the heck out of myself.

Reply:
Okay, I changed it to `theme.palette.primary.contrastText` and it's showing up as white now. Sorry for my dumbness.